src/QsCompiler/SyntaxProcessor/ScopeTools.fs

```diff
@@ -226,9 +233,12 @@ type SymbolTracker(globals: NamespaceManager, sourceFile, parent: QsQualifiedNam
             LocalVariableDeclaration<_>.New mut ((decl.Position, decl.Range), name, decl.Type, qDep)
             |> this.TryAddVariableDeclartion
 
+    /// <summary>
     /// Updates the quantum dependency of the local variable with the given name.
-    /// Throws an ArgumentException if no local variable with the given name is visible on the current scope.
-    /// Throws an InvalidOperationException if no scope is currently open, or the variable is immutable.
+    /// </summary>
+    /// <exception cref="ArgumentException"><paramref name="varName"/> is not visible in the current scope.</exception>
+    /// <exception cref="InvalidOperationException">No scope is currently open.
```

Comment:
```suggestion
    /// <exception cref="InvalidOperationException">No scope is currently open.</exception>
```

Sorry, it looks like I missed the closing `</exception>` tag in my last suggestion. I'm surprised the doc comment verification didn't cause the build to fail, though. It looks like it actually is working in Visual Studio (this comment has a red squiggly), but running `dotnet build` from the command line doesn't catch the error.

Do you know why this might be happening - maybe the `<OtherFlags>--warnon:3390</OtherFlags>` tag isn't working with `dotnet build` for some reason?

Reply:
Hmm that's a great question. Does passing the warning flag on the command line work? If not, is it possible that a version of F# Tools older than 11.0.0 is being used under the hood? I also wonder what version is used in CI.

I should be able to poke around at some point over the next day or two. Thanks for your patience on working through this PR!

Reply:
Oh, I forgot that only F# from .NET 5 can do the validation. :) I think that would explain it - I'm using Visual Studio 16.8 which comes built-in with .NET 5, but `dotnet` on the command line follows the global.json file which pins the version to .NET Core 3.1. The CI is also using 3.1. So it sounds like XML comment validation will probably start working when we update those to .NET 5.

Reply:
Ah, that's right! I forgot that I updated the config for .NET 5 when I was testing. Yes, it should start working as soon as that upgrade is complete :)